samples/boards/st/power_mgmt/standby/boards/stm32l4-standby.overlay

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2025 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	cpus {
+		cpu@0 {
+			cpu-power-states = <&stop0 &stop1 &stop2 &standby>;
+		};
+
+		power-states {
+			standby: state3 {
```

Comment:
To be moved to .dtsi file

Reply:
Ok, I created a dedicated commit for updating L4 DTSI file.